test/csuite/Makefile.am

```diff
@@ -57,6 +57,9 @@ noinst_PROGRAMS += test_wt3135_search_near_collator
 test_wt3184_dup_index_collator_SOURCES = wt3184_dup_index_collator/main.c
 noinst_PROGRAMS += test_wt3184_dup_index_collator
 
+test_wt3338_partial_update_SOURCES = wt3338_partial_update/main.c
+noinst_PROGRAMS += test_wt3338_partial_update
+
```

Comment:
I looked at all the tests below and nothing specifically tests modify with logging enabled that I can tell.  The csuite program and the python program never use logging AFAICT.  Am I missing it somewhere?  Also, it would be good to have a new test section in `test_compat01.py` or even a new `test_compat02.py` that tests modify and compatibility, confirming that the new record never ends up in the old version log file.